### DISCUSSION TITLE
429: Email rate limit exceeded (Custom SMTP)

### DISCUSSION THREAD

**Discussion by Barefoot-Dev** [supabase/supabase#16209] [Questions] [answered]
I know the rate limit is 4 for the built-in service, but I'm using a custom SMTP provider (resend) and still hitting the rate limit of 4.

Has anyone else had this?

**Comment by GaryAustin1** [comment, marked as answer]
In the custom SMTP settings screen did you set a higher limit for emails per hour?


Edit.  The configuration for this moved to the Authentication settings tab under rate limits.

**Reply by Barefoot-Dev** [reply]
Huh, I didn't notice this setting... :facepalm:  Thanks

**Reply by SkeltalFlamingo** [reply]
where can I find this screen?

**Reply by jondashkyle** [reply]
@SkeltalFlamingo It’s in Authentication → Rate Limits. Also found this confusing fwiw!

**Reply by SkeltalFlamingo** [reply]
@jondashkyle , thanks so much! I literally spent an hour looking and couldn't find it. Yesterday I found it by accident. I'm glad you responded so hopefully someone else finds it faster than we did.

**Reply by davenalluri** [reply]
@GaryAustin1 , @jondashkyle , where can I configure this on self hosted version of Supabase? I am using a custom SMTP server to send mails, however unable to find the env variable to configure this rate limit. Could you please give the env variable name for the same?

**Reply by colinchadwick** [reply]
@davenalluri The env variable for that is `GOTRUE_RATE_LIMIT_EMAIL_SENT`, see https://github.com/supabase/gotrue.
